qutebrowser/mainwindow/tabwidget.py

```diff
@@ -79,6 +79,7 @@ def init_config(self):
         self.setTabPosition(position)
         tabbar.vertical = position in [QTabWidget.West, QTabWidget.East]
         tabbar.setSelectionBehaviorOnRemove(selection_behavior)
+        tabbar.set_font()
```

Comment:
The intention is to get changes to `tabs->favicon-scale` to recompute the favicon size, which is done based on the font size. I guess I'll just move that out to its own function, though.